clang/cmake/caches/Release.cmake

```diff
@@ -44,6 +44,19 @@ set(LLVM_RELEASE_ENABLE_LTO THIN CACHE STRING "")
 set(LLVM_RELEASE_ENABLE_PGO ON CACHE BOOL "")
 set(LLVM_RELEASE_ENABLE_RUNTIMES ${DEFAULT_RUNTIMES} CACHE STRING "")
 set(LLVM_RELEASE_ENABLE_PROJECTS ${DEFAULT_PROJECTS} CACHE STRING "")
+
+# This option enables linking stage2 clang statically with the runtimes
+# (libc++ and compiler-rt) from stage1.  In theory this will give the
```

Comment:
it looks like compiler-rt is linked against regardless of this flag, so perhaps worth dropping from the list here?

Reply:
@c-rhodes Where do you see this?

Reply:
sorry you can ignore me, I was looking at `STAGE1_RUNTIMES` being set regardless of this flag but of course that's only to build, the linking bit comes later.